guide/17-geocoding-guides/part6_custom_geocoders.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>In this section, we will talk about specifying a particular <code>Geocoder</code> instance when there are multiple available (<code>geocode(geocoder=)</code>) </p><p>The <code>geocoder</code> parameter is optional for the <code>geocode</code> function, representing whichever geocoder to be used. If not specified, the active GIS's first geocoder is used. We can see from the example below, that with different geocoders, the geocoded results for the same address might be different, due to the difference of spatial reference, geocoder coverage and precision, etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>